packages/wonder-blocks-modal/docs.md

```diff
@@ -0,0 +1,35 @@
+# Custom modals
+
+```js
```

Comment:
This section would benefit from a sentence or two explaining what this code is doing and why people might want to do this.

Reply:
I don't know this is still showing up.  I've changed this to `# Wrapping Modals` and added a comment.